templates/tableau-single-server-centos.template

```diff
@@ -293,7 +293,7 @@
         },
         "DefaultConfiguration": {
             "InstallationConfig": {
-                "TableauServerInstaller": "https://s3-us-west-2.amazonaws.com/tableau-quickstart/tableau-server.x86_64.rpm",
+                "TableauServerInstaller": "https://s3-us-west-2.amazonaws.com/tableau-quickstart/tableau-server-2018-1-1.x86_64.rpm",
```

Comment:
We should not be changing the installer name; otherwise we will need to update the templates every quarter.  I usually just drop the new installers and rename them to what the templates expect.

Reply:
I agree, but if I just drop the new installers and rename them to the existing name, the current CF templates will be broken for our customer. Maybe we have to ask Santiago to deploy the new templates and change back the installer name at the same time.

Reply:
discussed with Kitty. I'll rename the installers and revert this code back. We're fine with CF not working shortly.

Reply:
Do we say anywhere in CF what version of server will be deployed?

Reply:
Not explicitly, we just say it will install the "latest version of Tableau Server".